### PR TITLE
Added Cilex and Pimple support

### PR DESCRIPTION
**Pull request by AlexStansfield**
I realise this is for the older version, but this should add providers for Pimple and Cilex

### CODE REVIEW

File: src/Pimple/AwsServiceProvider.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+use Aws\Sdk;
+use Guzzle\Common\Event;
+use Guzzle\Service\Client;
+
+/**
+ * AWS SDK for PHP service provider for Pimple based applications
+ */
+class AwsServiceProvider
+{
+    const VERSION = '2.0.2';
```
Comment (cjyclaire) [inline src/Pimple/AwsServiceProvider.php]: In order to tag this in a release, version number needs to be '2.1.0'

**Comment by cjyclaire** [comment]
@AlexStansfield , thanks for the PR! It looks good to me. Yet in order to tag a release for it, a minor bump needs to happen on v2 branch.

**Comment by AlexStansfield** [comment]
@cjyclaire done :)

**Comment by AlexStansfield** [comment]
Great, many thanks

**Pull request merged**